changes for icarus: add an option to extract pulse start time

drinkingkazu opened the pull request.

Commit:
changes for icarus: add an option to extract pulse start time

FNALbuild commented:
A new Pull Request was created by  @drinkingkazu (Drinking Kazu) for develop.

It involves the following packages:

larana


@LArSoft/level-1-managers, @LArSoft/level-2-managers can you please review it and eventually sign? Thanks.

cms-bot commands are listed <a href="https://cdcvs.fnal.gov/redmine/projects/larsoft/wiki/Pull_request_comments_that_trigger_CI_actions">here</a>

FNALbuild commented:
The code-checks are being triggered in jenkins.

FNALbuild commented:
+code-checks  
 for details see 
 https://buildmaster.fnal.gov/buildmaster/job/GithubIntegration/job/run-pr-code-checks//905/console

lgarren commented:
trigger build

FNALbuild commented:
The tests are being triggered in jenkins.

FNALbuild commented:
+LArSoft tests OK on slf7 for c7:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10970&builds= 
 +build

FNALbuild commented:
+LArSoft tests OK on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10969&builds= 
 +build

FNALbuild commented:
+LArSoft tests OK on slf7 for e20:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10971&builds= 
 +build

FNALbuild commented:
-lariat failed on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:lariat/view_builds/index?offset=0&builds=lariat_ci/5355&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10969&builds=

FNALbuild commented:
+argoneut tests OK on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:argoneut/view_builds/index?offset=0&builds=argoneut_ci/5583&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10969&builds=

FNALbuild commented:
-sbnd tests warning with build warning on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:sbnd/view_builds/index?offset=0&builds=sbnd_ci/6207&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10969&builds=

FNALbuild commented:
-icarus tests failed on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:icarus/view_builds/index?offset=0&builds=icarus_ci/1724&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10969&builds=

FNALbuild commented:
-dune tests warning on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:dune/view_builds/index?offset=0&builds=dune_ci/11070&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10969&builds=

FNALbuild commented:
-uboone tests warning on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:uboone/view_builds/index?offset=0&builds=uboone_ci/6027&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10969&builds=

knoepfel commented:
Will approve once we have L2 approval.

SFBayLaser commented:
+1

FNALbuild commented:
This pull request is fully signed and it will be merged to develop and built in the next LArSoft release after it passes the integration tests. This pull request will now be reviewed by the release team before it's merged.)

knoepfel commented on larana/OpticalDetector/OpHitFinder/OpHitAlg.cxx:
Shouldn't the backwards-compatible change be:
```diff
double absTime = timeStamp + clocksData.OpticalClock().TickPeriod() *
- (use_start_time ? pulse.t_start : pulse.t_end);
+ (use_start_time ? pulse.t_start : pulse.t_max);
```

drinkingkazu commented on larana/OpticalDetector/OpHitFinder/OpHitAlg.cxx:
ah yes! Sorry for this mistake in rush.

drinkingkazu committed:
Update OpHitAlg.cxx

Addressing Kyle's comment https://github.com/LArSoft/larana/pull/10/files/1f80340477bec36c7e9e7bd05bb438bc54a4d6f7#r599676003

FNALbuild commented:
Pull request #10 was updated. @LArSoft/level-1-managers, @LArSoft/level-2-managers can you please check and sign again.

FNALbuild commented:
The code-checks are being triggered in jenkins.

FNALbuild commented:
+code-checks  
 for details see 
 https://buildmaster.fnal.gov/buildmaster/job/GithubIntegration/job/run-pr-code-checks//906/console

lgarren commented:
trigger build

FNALbuild commented:
The tests are being triggered in jenkins.

knoepfel commented on larana/OpticalDetector/OpHitFinder/OpHitAlg.cxx:
No problem, Kazu.

FNALbuild commented:
+LArSoft tests OK on slf7 for c7:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10977&builds= 
 +build

FNALbuild commented:
+LArSoft tests OK on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10976&builds= 
 +build

FNALbuild commented:
+LArSoft tests OK on slf7 for e20:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10978&builds= 
 +build

FNALbuild commented:
-lariat failed on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:lariat/view_builds/index?offset=0&builds=lariat_ci/5357&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10976&builds=

FNALbuild commented:
+argoneut tests OK on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:argoneut/view_builds/index?offset=0&builds=argoneut_ci/5585&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10976&builds=

FNALbuild commented:
-sbnd tests warning on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:sbnd/view_builds/index?offset=0&builds=sbnd_ci/6212&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10976&builds=

FNALbuild commented:
-icarus tests failed on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:icarus/view_builds/index?offset=0&builds=icarus_ci/1726&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10976&builds=

FNALbuild commented:
-uboone tests warning on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:uboone/view_builds/index?offset=0&builds=uboone_ci/6029&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10976&builds=

FNALbuild commented:
-dune tests failed on slf7 for e19:prof 
 for details see 
 https://lar-ci-history.fnal.gov/LarCI/app/ns:dune/view_builds/index?offset=0&builds=dune_ci/11072&builds= 
 for details of the parent CI build see 
 https://lar-ci-history.fnal.gov/LarCI/app/view_builds/index?offset=0&builds=lar_ci/10976&builds=

lgarren commented:
It looks like the CI was OK.  There were some hiccups due to work on the database server today.

lgarren commented:
approve

FNALbuild commented:
This pull request is fully signed and it will be merged to develop and built in the next LArSoft release after it passes the integration tests.
